### ISSUE TITLE
After upgrade Grafana from v7.3.5 to v8.0.3 Grafana using Helm Chart v6.13.6 does not show my Dashboards and Configurations anymore running on Kubernetes v1.20.8

### ISSUE THREAD

**Issue by fhperuchi** [grafana/helm-charts#535]
I'm using an `existingClaim` named `grafana-data` that already exists on Kubernetes cluster and the data from the old version 7.3.5:

![image](https://user-images.githubusercontent.com/4641123/123835737-490fa200-d8df-11eb-8c68-47c580315212.png)

![image](https://user-images.githubusercontent.com/4641123/123834815-35177080-d8de-11eb-96f3-668a2cf8eedc.png)

![image](https://user-images.githubusercontent.com/4641123/123834900-49f40400-d8de-11eb-9bf8-c99106e34414.png)

![image](https://user-images.githubusercontent.com/4641123/123834598-f71a4c80-d8dd-11eb-95f1-6a29cfb8754b.png)

Chart `persistence` configuration:
```
persistence:
  type: pvc
  enabled: true
  storageClassName: gp2-retain
  accessModes:
    - ReadWriteOnce
  size: 2Gi
  # annotations: {}
  finalizers:
    - kubernetes.io/pvc-protection
  # selectorLabels: {}
  # subPath: ""
  existingClaim: grafana-data
```

Grafana .ini configuration file:
```
bash-5.1$ cat /etc/grafana/grafana.ini 
[analytics]
check_for_updates = true
[auth.github]
allow_sign_up = true
allowed_organizations = *******
api_url = https://api.github.com/user
auth_url = https://github.com/login/oauth/authorize
client_id = ******************
client_secret = ******************
enabled = true
root_url = https://grafana-xxxxxx.xxx.com
scopes = user:email,read:org
token_url = https://github.com/login/oauth/access_token
[grafana_net]
url = https://grafana.net
[log]
mode = console
[paths]
data = /var/lib/grafana/data
logs = /var/log/grafana
plugins = /var/lib/grafana/plugins
provisioning = /etc/grafana/provisioning
[server]
root_url = https://grafana-sandbox.pathable-ci.com
```

Kubernetes cluster version:
```
$ kubectl version --short 
Server Version: v1.20.8
```

Could you help please?

**Comment by ralph089** [comment]
Hit the same issue

**Comment by vitaliyf** [comment]
Was it fixed by #450 (and #478) for you?

Try "helm template ... | grep -A1 GF_PATHS_DATA" to check that the value matches where your grafana.db is located.

**Comment by fhperuchi** [comment]
same as this: https://github.com/grafana/helm-charts/issues/478
